yarn-project/sequencer-client/src/block_builder/circuit_block_builder.ts

```diff
@@ -274,14 +307,16 @@ export class CircuitBlockBuilder implements BlockBuilder {
       this.validateTrees(rootOutput),
       this.validateRootTree(rootOutput, MerkleTreeId.CONTRACT_TREE_ROOTS_TREE, 'Contract'),
       this.validateRootTree(rootOutput, MerkleTreeId.PRIVATE_DATA_TREE_ROOTS_TREE, 'PrivateData'),
+      this.validateRootTree(rootOutput, MerkleTreeId.L1_TO_L2_MESSAGES_ROOTS_TREE, 'L1ToL2Message'),
+      this.validateTree(rootOutput, MerkleTreeId.L1_TO_L2_MESSAGES_TREE, 'L1ToL2Message'),
     ]);
   }
 
   // Helper for validating a roots tree against a circuit simulation output
   protected async validateRootTree(
```

Comment:
Should we call this `validateHistoricRootsTree` instead for clarity? When there are public historic roots we can also check those.